scripts/token-hud-extension.js

```diff
@@ -279,34 +326,34 @@ class TokenHudExtension {
             html.find('.control-icon.target').wrap(divTokenHudExt);
 
             // Corruption and Sin
-            let hudCorruption = $(`<div class="control-icon tokenhudicon left" title="` + TooltipCorruption + `"><i class="fas fa-bahai">&nbsp;` + corruption + `</i></div>`);
+            let hudCorruption = $(`<div class="control-icon tokenhudicon left corruption" title="` + TooltipCorruption + `"><i class="fas fa-bahai">&nbsp;` + corruption + `</i></div>`);
```

Comment:
You've added class 'corruption' to this div, but not added similar to other hud divs. how is it used?